tools/witx/src/lib.rs

```diff
@@ -17,13 +21,14 @@ pub use ast::{
     ModuleDefinition, ModuleEntry, ModuleImport, ModuleImportVariant, StructDatatype, StructMember,
     UnionDatatype, UnionVariant,
 };
+pub use io::{Filesystem, MockFs, WitxIo};
 pub use lexer::LexError;
 pub use parser::{DeclSyntax, ParseError};
+pub use render::{Render, SExpr as RenderSExpr};
```

Comment:
Is `RenderSExpr` used anywhere? Renaming things in imports like this has its uses, but I also find it confusing, so I'm looking for what's motivating it here :-).

Reply:
It is not used anywhere under that name. When I renamed the import I was thinking it would disambiguate it from `sexpr::SExpr`, which has different types (&strs and locations) in the leaf nodes. But that type isn't publicly exposed by this library, so, perhaps I should just export this one as SExpr. You're correct that it isnt worth any potential confusion.